dali/operators/reader/nvdecoder/sequencewrapper.h

```diff
@@ -37,8 +37,10 @@ struct SequenceWrapper {
  public:
   SequenceWrapper() = default;
 
-  void initialize(int count, int height, int width, int channels, DALIDataType dtype) {
+  void initialize(int count, int max_count, int height, int width, int channels,
+                  DALIDataType dtype) {
     this->count = count;
+    this->max_count = max_count;
```

Comment:
You can probably reserve timestamps to max_count in line 50, I see `prefetched_video.timestamps.resize(count_);` in dali/operators/reader/video_reader_op.h

Reply:
Done